lms/services/organization.py

```diff
@@ -78,4 +115,4 @@ def auto_assign_organization(
 def service_factory(_context, request) -> OrganizationService:
     """Get a new instance of OrganizationService."""
 
-    return OrganizationService(db_session=request.db)
+    return OrganizationService(db_session=request.db, region=request.region)
```

Comment:
It's the way we tend to do things, but it seems a shame that the region is only available on the request rather than in some more global way. We know this before any request is made. The downside of this is, a service which is otherwise a global instance might be forced to become a request instance just to get this.

I wonder if there is any other peg to hang this off?

Reply:
Having to access global configuration through the request is a bit strange but I reckon finding an alternative would be too much pyramid fighting for this PR.